core/server/monitor.go

```diff
@@ -300,6 +300,10 @@ func reloadConfig() {
 	proxy.JSSecret = domains.Config.Proxy.Secrets["javascript"]
 	proxy.CaptchaSecret = domains.Config.Proxy.Secrets["captcha"]
 
+	proxy.IdleTimeout = domains.Config.Proxy.Timeout.Idle
```

Comment:
Like you stated in your comment, these will be 0 if not provided. Ideally the proxy should have default values that (almost) never need changing. In the current state, the proxy would always instantly time out, if you don't edit the json to include your variables. I'd be nicer if the default values are those that they are right now but can be changed if provided in the json

Reply:
Should i default within the config init?

Reply:
Defaults have been added. Please review my last commit.